RecoEgamma/EgammaIsolationAlgos/src/EleTkIsolFromCands.cc

```diff
@@ -91,9 +111,13 @@ EleTkIsolFromCands::Output EleTkIsolFromCands::operator()(const reco::TrackBase&
   auto const& preselectedTracks = getPreselectedTracks(isBarrelElectron);
   auto const& cuts = isBarrelElectron ? cfg_.barrelCuts : cfg_.endcapCuts;
 
-  for (auto& trk : preselectedTracks) {
-    if (passMatchingToElectron(trk, cuts, eleEta, elePhi, eleVz)) {
-      ptSum += trk.pt;
+  for (auto const& trk : preselectedTracks) {
+    const float dR2 = reco::deltaR2(eleEta, elePhi, trk.get<Eta>(), trk.get<Phi>());
+    const float dEta = trk.get<Eta>() - eleEta;
```

Comment:
typical mess of mixed precision

Reply:
Thanks for the reminder! I didn't change this so far, because I didn't want to risk even minor precision differences in the output. Would be hard to disentangle from possible mistakes I made in my code. Now that the comparisons showed no reco differences, I'm happy to address this.